pages/data/_content.yaml

```diff
@@ -1,25 +1,20 @@
--
-  url: /data/faers/
+- url: /data/faers/
   title: FAERS
   description:
-    - The FDA Adverse Event Reporting System (FAERS) is a database that contains information on adverse event and medication error reports submitted to FDA.
-    - See the [FAERS web site](http://www.fda.gov/Drugs/GuidanceComplianceRegulatoryInformation/Surveillance/AdverseDrugEffects/ucm082193.htm) for download information.

```

Comment:
@beardedfinch Removal of this para intentional?

Reply:
Yup, I fixed some formatting on that data page.